protobuf-definitions/proto/mlagents/envs/communicator_objects/observation.proto

```diff
@@ -8,8 +8,15 @@ enum CompressionTypeProto {
     PNG = 1;
```

Comment:
Maybe NONE and PNG should be changed to `NO_COMPRESSION` and `PNG_COMPRESSION` to avoid the bug you see with `NONE` ?

Reply:
The problem isn't with NONE, it looks like it's any enum in protobuf <3.8. See https://github.com/protocolbuffers/protobuf/issues/6028.

That being said, if you think those are better enum names, we can change them.

Reply:
Alternatively, we can also update the min protobuf version to 3.8.0